src/Appwrite/Extend/Exception.php

```diff
@@ -106,9 +106,16 @@ class Exception extends \Exception
     public const STORAGE_INVALID_CONTENT_RANGE     = 'storage_invalid_content_range';
     public const STORAGE_INVALID_RANGE             = 'storage_invalid_range';
 
+    /** VCS */
+    public const INSTALLATION_NOT_FOUND              = 'installation_not_found';
+    public const REPOSITORY_NOT_FOUND                = 'repository_not_found';
```

Comment:
We need proper prefix and to follow existing naming patterns.

Reply:
We removed VCS prefixed from everywhere. If its about non-appwrite repository, it's PROVIDER_REPOSITORY_NOT_FOUND